### PR TITLE
Add nft collection example

### PR DESCRIPTION
**Pull request by Thoralf-M**

### CODE REVIEW

File: examples/nft_collection/01_mint_collection_nft.rs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+use iota_wallet::{account_manager::AccountManager, NftOptions, Result};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses dotenv, which is not safe for use in production
+    dotenv().ok();
+
+    let nft_collection_size = 15;
+    // Set this to the NFT id from the mint_issuer_nft example
+    let issuer_nft_id = NftId::from_str("0x13c490ac052e575cffd40e170c2d46c6029b8b68cdf0e899b34cde93d2a7b28a")?;
```
Comment (thibault-martinez) [inline examples/nft_collection/01_mint_collection_nft.rs:24]: How do they get it from mint_issuer_nft?

Comment (Thoralf-M) [reply examples/nft_collection/01_mint_collection_nft.rs:24]: They have to check the balance for the new NFT ID, get it form the explorer or whatever :trollface:
Should I add some code to print the NFT ID?

Comment (thibault-martinez) [reply examples/nft_collection/01_mint_collection_nft.rs:24]: That would be very nice 🙏🏻

Comment (Thoralf-M) [reply examples/nft_collection/01_mint_collection_nft.rs:24]: Output looks like this now:
```
Transaction: 0x2ac532698e01cd35cf815adb9cea1ed3132c17377bed5a2d503980d030c82010.
Block sent: https://api.testnet.shimmer.network/api/core/v2/blocks/0x4e87586cf74959e6f3b640b246b4640bf97248b367246b5163d747d641dd542a.
New minted NFT id: 0xb2e0c6bb23b27c690245208d91bd7b185072d45084881e8584cb59e36435d61e
```

**Review by thibault-martinez — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**